bin/run-wer-automation.sh

```diff
@@ -25,7 +14,7 @@ python -u DeepSpeech.py \
   --dropout_rate 0.30 \
   --default_stddev 0.046875 \
   --checkpoint_step 1 \
-  --export_dir "${ds_dataroot}/exports/`git rev-parse --short HEAD`" \
+  --checkpoint_dir "${COMPUTE_KEEP}" \
+  --export_dir "${COMPUTE_RESULTS}" \
+  --wer_log_pattern "GLOBAL LOG: logwer('${COMPUTE_ID}', '%s', '%s', %f)"\
   "$@"
-
-ln -sf $ds_export_dir $ds_export_dir/../latest
```

Comment:
The exports are now tied to and kept in their run directories. There is not one latest anyhow, as we explore different solutions. However: We could provide a script to pick (bsed on some criteria) a latest one from the run directories...